README.md

```diff
@@ -82,7 +93,7 @@ Prism can be forced to return different HTTP responses by specifying the status
 string:
 
 ```bash
-http GET http://127.0.0.1:4010/pets/123?__code=404
+$ curl http://127.0.0.1:4010/pets/123?__code=404
```

Comment:
Any time you are writing curl examples, don't include the `$`. It often gets copied by users and then they get the annoying this "command cannot be found" error.

Reply:
Very good point! Removing it!